src/client.rs

```diff
@@ -627,36 +626,22 @@ mod tests {
         assert!(sample_set_1 != sample_set_2);
     }
 
-    #[test]
-    #[should_panic(expected = "Rank cannot be greater than or equal to world size")]
-    fn test_broken_ranks() {
```

Comment:
moving this to the different backends, instead of that being part of the main config